src/registration/locale/de/LC_MESSAGES/django.po

```diff
@@ -7,413 +7,412 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2015-07-22 00:53-0500\n"
+"POT-Creation-Date: 2016-04-26 14:54+0800\n"
 "PO-Revision-Date: 2015-07-28 09:16+0200\n"
+"Last-Translator: Patrick Husi <patrick.husi@arpage.ch>\n"
+"Language-Team: \n"
 "Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"Last-Translator: Patrick Husi <patrick.husi@arpage.ch>\n"
-"Language-Team: \n"
 "X-Generator: Poedit 1.8.2\n"
 
-#: src/registration/forms.py:66
+#: admin/__init__.py:262
```

Comment:
Well I am not really sure but why `src/registration/` prefix is omitted? Do you have idea? Maybe the version difference of the django (then it's ok)?